index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
+    <title>Banner</title>
+  </head>
+  <body style="background-color: black">
+    <main>
+      <section class="banner-section">
+        <div class="container-banner">
+          <h2 class="main-title">Join the Escrow Manager Program</h2>
+          <div class="banner-wrapper">
+            <div class="left-section">
+              <img
+                class="banner-image"
+                src="https://www.chainpals.io/assets/images/guest/png/become-escrow-manager.png"
+                alt="banner-image"
+              />
+            </div>
+            <div class="right-section">
+              <h2 class="title">Join the Escrow Manager Program</h2>
+              <p class="description">
+                Become a Chainpals Escrow Manager and start earning passive
+                income by hodling EM NFT.
+              </p>
+              <div class="button-wrapper">
+                <a
+                  class="learnmore-link-button"
+                  href="https://www.chainpals.io/escrow-manager"
+                  >LEARN MORE</a
+                >
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="roadmap-section">
+        <div class="wrapper">
+          <div class="backgroundImg"></div>
```

Comment:
remove this need it with before after or background set @aeshapatel025

Reply:
chnages done